index.test.js

```diff
@@ -1446,4 +1467,36 @@ describe('Deploy to ECS', () => {
         expect(core.setFailed).toHaveBeenNthCalledWith(1, 'Failed to register task definition in ECS: Could not parse');
         expect(core.setFailed).toHaveBeenNthCalledWith(2, 'Could not parse');
     });
+
+    test('propagate service tags from service and enable ecs managed tags', async () => {
```

Comment:
Hi @guikcd, thanks for this PR! 
I believe we are missing unit tests that checks the `run-task-tags` functionality. Would it also be possible to add unit tests for the `run-task-tags` parameter/functionality as part of existing or as a new test.

Reply:
I've modified the existing test `run task with options` to verify run-task-tags.